index.js

```diff
@@ -0,0 +1,42 @@
+const checkValidNum = (value) => {
+  if (value === '') {
+    return false
+  } else {
+    return !Number.isNaN(Number(value)) || value === null && value !== undefined
+  }
+}
+
+const findNums = (arr) => {
+  return (arr.filter(checkValidNum))
+}
+
+const findOperator = (arr) => {
+  for (let i = 0; i < (arr.length); i++) {
+    if (arr[i] === '+' || arr[i] === '-' || arr[i] === '*' || arr[i] === '/') {
+      return arr[i]
+    }
+  }
+
+  return [NaN]
+}
```

Comment:
Searching for the operator at any position will cause you problems. `[10, 15, '+']` should be NaN since it is not a valid calculation. Order matters.